leapp/reporting/__init__.py

```diff
@@ -245,6 +245,10 @@ def __init__(self, value=None):
         self._value = {'type': 'command', 'context': value}
 
     def __repr__(self):
+        # NOTE(ivasilev) As filenames can contain non-ascii characters let's deal with it properly.
```

Comment:
a nitpick, filenames can be confusing here. I would stick with "the string" or something generic. The issue does not stick just with filenames. It's general problem. The same in other notes.
 
```suggestion
        # NOTE(ivasilev) As the string can contain non-ascii characters let's deal with it properly.
```